src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/commerce/catalogpage/customfooterlibs.html

```diff
@@ -13,7 +13,7 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 */-->
-<sly data-sly-use.clientlib="/libs/granite/sightly/templates/clientlib.html">
-    <sly data-sly-call="${symbol_dollar}{clientlib.js @ categories='${appId}.base'}"/>
+<sly data-sly-use.clientlib="core/wcm/components/commons/v1/templates/clientlib.html">
+    <sly data-sly-call="${symbol_dollar}{clientlib.js @ categories='${appId}.base', async=true}"/>
```

Comment:
Could we end up in a similar setup as the one described at [0], e.g. one clientlib that is not compatible with asynchronous loading?

[0] https://github.com/adobe/aem-core-wcm-components/issues/2040

Reply:
This clientlib contains only the scripts from the WCM Core Components and should not have any issues with `async` 

https://github.com/adobe/aem-project-archetype/blob/644470b6b22dc97a0a1fb1a3f533617935f208ad/src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/clientlibs/clientlib-base/.content.xml